src/assets/scripts/client/navigationLibrary/Route/RouteModel.js

```diff
@@ -135,15 +135,28 @@ export default class RouteModel extends BaseModel {
     }
 
     /**
-     * Verify that a routeCode has exactly 3 segments
+     * Verify that a routeString has exactly 3 segments
      *
      * @for RouteModel
      * @method _isValidRouteCode
-     * @param routeCode {string}
+     * @param routeString {string}
      * @return {boolean}
      * @private
      */
-    _isValidRouteCode(routeCode) {
-        return routeCode.split(SEGMENT_SEPARATION_SYMBOL).length === MAXIMUM_ROUTE_SEGMENT_LENGTH;
+    _isValidRouteCode(routeString) {
+        return RouteModel.isProcedureRouteString(routeString);
     }
```

Comment:
I like those terms, I'm stealing them for the issue I just added #262 to cover this upgrade. 😉 

Right now, the `RouteModel` is *_only_* acting as a `ProcedureRouteSegmentModel`.  Thats it.  If a routeString is not a procedure, it doesn't even get into the class and we handle it via other means.